csympy/lib/csympy_wrapper.pyx

```diff
@@ -249,6 +249,31 @@ cdef class Symbol(Basic):
         import sympy
         return sympy.Symbol(str(deref(X).get_name().decode("utf-8")))
 
+
+def symbols(names, *args):
```

Comment:
Normally, one would call `symbols` using named arguments. For example:
```python
x, y, z = symbols('x,y,z', real=True)
```
We are not supporting this at the moment but I think it will be better to use `**args` instead of `*args`.